.golangci.yml

```diff
@@ -0,0 +1,30 @@
+run:
+  deadline: 5m
+
+linters:
```

Comment:
these linters and their settings match what was previously specified in the precommit script